src/components/nodes/pool/pool.vue

```diff
@@ -174,7 +174,7 @@ export default {
       lanes.push(this.pushNewLane());
 
       await Promise.all(lanes);
-      this.$emit('set-shape-stacking', this.shape);
+      this.$emit('set-shape-stacking', this.shape, this.graph);
```

Comment:
The use of `this.graph` is also acting like a flag because of the `if` statement in `bringFlowsToFront`. Do you think it would be cleaner to leave the `ensureShapeIsNotCovered` function the same, and loop through sequence flows here? Something like:

```js
this.$emit('set-shape-stacking', this.shape);
this.graph.getLinks().forEach(link => {
  this.$emit('set-shape-stacking', link);
});
```